js/web/lib/onnxjs/instrument.ts

```diff
@@ -70,6 +71,12 @@ export interface Logger {
    * @param config the config object to indicate the logger's behavior
    */
   set(category: string, config: Logger.Config): void;
+
+  /**
+   * Set the logger's behavior from ort-common env
+   * @param env the env used to set logger. Currently only setting loglevel is supported through Env.
+   */
+  setWithEnv(env: Env): void;
```

Comment:
I think you can just use setWithEnv(loggingLevel: string) here because you don't care the other field in env

Reply:
we might add other fields to env in the future, this will just prevent any api change or override.